zhaquirks/inovelli/__init__.py

```diff
@@ -245,66 +196,224 @@ def handle_cluster_request(
             return
 
 
-VZM35SN_REMOVES = [
-    0x0012,
-    0x0013,
-    0x0014,
-    0x0019,
-    0x001A,
-    0x0034,
-    0x0064,
-    0x007D,
-    0x0105,
-]
+class InovelliVZM31SNCluster(InovelliCluster):
+    """Inovelli VZM31-SN custom cluster."""
+
+    name = "InovelliVZM31SNCluster"
+
+    attributes = InovelliCluster.attributes.copy()
+
```

Comment:
Can you also remove this empty line between `attributes = ...` and `attributes.update(...` everywhere?
(To follow what other quirks do)